ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -249,6 +249,11 @@ class TransactionSummaryScreen extends React.Component<Props> {
       .mapNullable(_ => _.enteBeneficiario)
       .map(formatTextRecipient);
 
+    const organizationFiscalCode: string = pot
+      .toOption(potVerifica)
+      .mapNullable(_ => _.enteBeneficiario?.identificativoUnivocoBeneficiario)
+      .getOrElse("-");
```

Comment:
```suggestion
      .getOrElse(rptId.organizationFiscalCode);
```
In the very unlucky case of a missing fiscal code, we could fallback to the original one. What do you think?

Reply:
Yes, I guess it is a good idea
I added a comment to explain that logic
changed here https://github.com/pagopa/io-app/pull/3946/commits/da2beeddfbc2c5d964e1aca3f0f18e4cb27e480b